### PR TITLE
Added com.github.liferooter.textpieces.json

### PR DESCRIPTION
**Pull request by liferooter**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

**Comment by LeandroStanger** [comment]
bot, build com.github.liferooter.textpieces

**Comment by liferooter** [comment]
Oh, sorry, I've changed manifest just before submission and didn't tried to build with new one. I'll fix it and submit yet another time

**Comment by LeandroStanger** [comment]
bot, build com.github.liferooter.textpieces

**Comment by amikha1lov** [comment]
bot, build com.github.liferooter.textpieces

**Comment by amikha1lov** [comment]
bot, build com.github.liferooter.textpieces

File: com.github.liferooter.textpieces.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+  - name: libhandy
+    buildsystem: meson
+    builddir: true
+    config-opts:
+      - -Dexamples=false
+      - -Dtests=false
+    sources:
+      - type: git
+        url: https://gitlab.gnome.org/GNOME/libhandy.git
```
Comment (nedrichards) [inline com.github.liferooter.textpieces.yaml:30]: this needs to refer to a specific commit, tag or release.

**Comment by nedrichards** [comment]
Other than pinning the libhandy release this lgtm

**Comment by LeandroStanger** [comment]
bot, build com.github.liferooter.textpieces

**Pull request closed without merging**